### PR TITLE
fix: Update message format

### PR DESCRIPTION
**Pull request by raji-unni**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

**Comment by ekremney** [comment]
@raji-unni from the formatting & logic perspective, it looks solid 👍 One thing we need is the `slackClient#postMessage` assertions in the test cases. Since the message formatting logic is quite complex for the experimentation audit, we need to make sure that we have assertions for `slackClient#postMessage` method so that someone else wouldn't break the formatting here with a change.

**Comment by raji-unni** [comment]
> @raji-unni from the formatting & logic perspective, it looks solid 👍 One thing we need is the slackClient#postMessage assertions in the test cases. Since the message formatting logic is quite complex for the experimentation audit, we need to make sure that we have assertions for slackClient#postMessage method so that someone else wouldn't break the formatting here with a change.

**Comment by raji-unni** [comment]
I have incorporated this test case as suggested.

File: src/experimentation/handler.js [hunk 1]
```diff
@@ -33,33 +33,84 @@ export function buildExperimentationSlackMessage(url, auditResult) {
   blocks.push(section({
     text: markdown(`For *${url}*, ${Object.keys(groupedData).length} experiments have been run in the *last week*.\n More information is below :`),
   }));
-
   Object.entries(groupedData).forEach(([key, value]) => {
+    const numOfVariants = value.length;
+    const duration = (date1, date2) => Math.abs(date2 - date1) / (1000 * 60 * 60 * 24);
```
[the remark this answers is not in the document]
Comment (raji-unni) [reply src/experimentation/handler.js:38]: completed the change.

File: src/experimentation/handler.js [hunk 2]
```diff
@@ -33,33 +33,84 @@ export function buildExperimentationSlackMessage(url, auditResult) {
...
+    const variantConfidence = [];
+    let totalVariantConversions = 0;
+    let totalVariantExperimentations = 0;
+    for (let i = 0; i < value.length; i += 1) {
+      const time95 = new Date(value[i].time95);
+      const time5 = new Date(value[i].time5);
+      variantDuration.push(duration(time95, time5));
+      variantConfidence.push({ vName: value[i].variant, vConfidence: value[i].p_value });
+      let vConversions = 0;
+      if (value[i].variant_conversions === null) {
```
[the remark this answers is not in the document]
Comment (raji-unni) [reply src/experimentation/handler.js:49]: completed the change. This is good learning for me. Thank you !

File: src/experimentation/handler.js [hunk 3]
```diff
@@ -33,33 +33,84 @@ export function buildExperimentationSlackMessage(url, auditResult) {
...
+        vConversions = 0;
+      } else vConversions = Number(value[i].variant_conversions);
+      let vExperimentations = 0;
+      if (value[i].variant_experimentations === null) {
```
[the remark this answers is not in the document]
Comment (raji-unni) [reply src/experimentation/handler.js:53]: done

File: src/experimentation/handler.js [hunk 4]
```diff
@@ -33,33 +33,84 @@ export function buildExperimentationSlackMessage(url, auditResult) {
...
-
-    const variantstats = [];
-    for (let i = 0; i < Math.min(3, value.length); i += 1) {
-      const score = (confidence) => {
-        if (confidence < 0.005) {
-          return `${confidence} *(highly significant)*`;
-        }
-        if (confidence < 0.05) {
-          return `${confidence} *(significant)*`;
+    const countformat = new Intl.NumberFormat('en-US', { maximumSignificantDigits: 2 });
```
[the remark this answers is not in the document]
Comment (raji-unni) [reply src/experimentation/handler.js:65]: Done.

**Review by ekremney — APPROVED** [review]

**Pull request merged**